avocado/plugins/spawners/podman.py

```diff
@@ -246,14 +250,6 @@ async def _create_container_for_task(
 
     async def spawn_task(self, runtime_task):
         self.create_task_output_dir(runtime_task)
-        podman_bin = self.config.get("spawner.podman.bin")
-        try:
-            # pylint: disable=W0201
-            self.podman = Podman(podman_bin)
-        except PodmanException as ex:
-            runtime_task.status = str(ex)
```

Comment:
I think this change doesn't work properly. When I run this on clean environment without podman, avocado crashes.

```
avocado run --nrunner-spawner=podman /bin/true
JOB ID     : 9facb7cce0be5dcec7393e7dfda831b03ebf2202
JOB LOG    : /root/avocado/job-results/job-2022-08-05T08.40-9facb7c/job.log

Avocado crashed: PodmanException: Podman binary /usr/bin/podman is not available on the system.
Traceback (most recent call last):

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/core/job.py", line 630, in run_tests
    summary |= suite.run(self)

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/core/suite.py", line 308, in run
    return self.runner.run_suite(job, self)

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/plugins/runner_nrunner.py", line 322, in run_suite
    loop.run_until_complete(

  File "/usr/lib64/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()

  File "/usr/lib64/python3.10/asyncio/tasks.py", line 408, in wait_for
    return await fut

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/core/task/statemachine.py", line 348, in run
    await self.start()

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/core/task/statemachine.py", line 268, in start
    start_ok = await self._spawner.spawn_task(runtime_task)

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/plugins/spawners/podman.py", line 255, in spawn_task
    major, minor, _ = await self.python_version

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/plugins/spawners/podman.py", line 174, in python_version
    result = await self.podman.get_python_version(image)

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/plugins/spawners/podman.py", line 114, in podman
    self._podman = Podman(podman_bin)

  File "/root/.local/lib/python3.10/site-packages/avocado_framework-98.0-py3.10.egg/avocado/utils/podman.py", line 40, in __init__
    raise PodmanException(msg)

avocado.utils.podman.PodmanException: Podman binary /usr/bin/podman is not available on the system.

Please include the traceback info and command line used on your bug report
Report bugs visiting https://github.com/avocado-framework/avocado/issues/new
```